packages/build/src/github-repo.ts

```diff
@@ -60,7 +57,15 @@ export class GithubRepo {
         this.repo,
       );
 
-    return tags.find(({ commit }) => commit.sha === sha);
+    const sortedDraftTags = tags
+      .filter(t => t.name && t.name.startsWith(`v${releaseVersion}`) && t.name.match(/^v\d+\.\d+\.\d+-draft\.\d+/))
```

Comment:
```suggestion
      .filter(t => t.name && t.name.startsWith(`v${releaseVersion}-draft`) && t.name.match(/^v\d+\.\d+\.\d+-draft\.\d+/))
```

Otherwise this would match `v0.7.10` for a `releaseVersion` of `v0.7.1`?

Reply:
Thanks for spotting 👍